sdk/storage/azure-storage-file-datalake/setup.py

```diff
@@ -93,7 +93,7 @@
     install_requires=[
         "azure-core<2.0.0,>=1.0.0",
         "msrest>=0.6.10",
-        "azure-storage-blob~=12.0.0"
+        "azure-storage-blob>=12.0.0"
```

Comment:
@xiafu-msft : Can we change this to "<13.0.0,>=12.0.0" to match with other packages which has dependency with azure-storage-blob. Dependency with just version 12.0.0 is causing current build issue.
@lmazuel , @rakshith91 , @annatisch : Please let me know if you have any comments or okay with this change to update the dependency to "<13.0.0,>=12.0.0"

Reply:
<13.0.0,>=12.0.0 sounds good to me

Reply:
Also, would be great if you can start a live run for python-client and python-storage-tests and verify this will resolve the issue

Reply:
should be <13.0.0,>=12.0.0